src/wasm/wasm-binary.cpp

```diff
@@ -628,7 +628,7 @@ void WasmBinaryWriter::writeNames() {
   }
 
   // table names
-  if (wasm->table.exists && wasm->table.name.is()) {
+  if (wasm->table.exists && wasm->table.explicitName) {
```

Comment:
Renaming `explicitName` to `isNameExplicit` would help make it clear that this is a boolean and not the actual name.

Reply:
Sounds good.  I went with `hasExplicitName`